extensions/wikia/AdEngine/js/lookup/rubiconTargeting.js

```diff
@@ -0,0 +1,36 @@
+/*global define*/
```

Comment:
I wonder if we could move this file to directory called rubicon in lookup directory - similar thing we did for openX. Although I don't feel strong about it but it would be nice to have it consistent for all bidders. Either move this file or extract the openX helper file.